PyQt5-stubs/QtCore.pyi

```diff
@@ -1650,6 +1650,10 @@ class QObject(sip.wrapper):
 
     def __init__(self, parent: typing.Optional['QObject'] = ...) -> None: ...
 
+    @typing.overload  # type: ignore  # fixes issue #1
```

Comment:
I don't understand the comment here - what issue #1?

Reply:
Yeah that's a little weird isn't it? It's referring to the first issue in [here](https://github.com/stlehmann/PyQt5-stubs/blob/master/issues.md). I was just following the precedent that already existed in the repo.

Some examples: [1](https://github.com/stlehmann/PyQt5-stubs/blob/bd0fe7ce030e9c97fe046878351b34a8a7ec8de6/PyQt5-stubs/QtPrintSupport.pyi#L118), [2](https://github.com/stlehmann/PyQt5-stubs/blob/bd0fe7ce030e9c97fe046878351b34a8a7ec8de6/PyQt5-stubs/QtSql.pyi#L484), [3](https://github.com/stlehmann/PyQt5-stubs/blob/cd4148a3ca213a895ddbc3431247088dad047ae0/PyQt5-stubs/QtCore.pyi#L1941)

I can remove it if you think it's unnecessary

Reply:
The comment refers to the collection of common issues in https://github.com/stlehmann/PyQt5-stubs/blob/master/issues.md and adresses overload errors. Issue #1 adresses overload decorators. I wanted to keep track of common issues so they could be fixed upstream someday.

Reply:
Ah, right. I definitely support having such an issues.md (IIRC I even originally suggested that). I thought of GitHub issues when reading "Issue #1", which didn't make much sense. Probably fine to leave it as-is though.